codice/ddf
# DDF-4413 Migrate dropdown.map-settings.view.js to React

#### What does this PR do?
Migrates map settings dropdown to React.
#### Who is reviewing it? 

@BernardIgiri 
@Variadicism 
#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@djblue 
#### How should this be tested?

Verify that the map settings dropdown has the same functionality as before.
1. When the user chooses a coordinate system, a corresponding coordinate example text should be displayed. 
2. The user's preferences should also be updated to reflect the selection (can be verified by looking at location display in the bottom left of the map). Changes should persist after a refresh.
#### Any background context you want to provide?

#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Screenshots

![image](https://user-images.githubusercontent.com/4467636/50709253-6a4fea00-1024-11e9-820f-32d7c93dd30d.png)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
